addons/fix-uploaded-svgs/userscript.js

```diff
@@ -37,4 +37,11 @@ export default async function ({ addon, global, console }) {
     };
     return realFileReader;
   };
+  window.FileReader = pollutedFileReader;
+  addon.self.addEventListener("disabled", () => {
+    window.FileReader = originalFileReader;
+  });
+  addon.self.addEventListener("reenabled", () => {
+    window.FileReader = pollutedFileReader;
+  });
```

Comment:
This is a pattern that should always be avoided.
Functions should never be unpolluted, because two addons can pollute the same object in an unpredictable order. What would happen if one of them decides to unpollute the object?
This is the only addon that pollutes `FileReader`, but it's still a bad idea.
Use an if statement in the polluted function instead.